address_test.go

```diff
@@ -13,25 +13,32 @@ import (
 	"testing"
 
 	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcd/wire"
 	"github.com/btcsuite/btcutil"
 	"golang.org/x/crypto/ripemd160"
 )
 
 type CustomParamStruct struct {
+	Net              wire.BitcoinNet
 	PubKeyHashAddrID byte
 	ScriptHashAddrID byte
+	Bech32HRPSegwit  string
 }
 
 var CustomParams = CustomParamStruct{
-	PubKeyHashAddrID: 0x30, // starts with L
-	ScriptHashAddrID: 0x32, // starts with M
+	Net:              0xdbb6c0fb,
```

Comment:
Can you please add a comment here? Example below:
```go
Net:              0xdbb6c0fb,  // litecoin mainnet HD version bytes
```

Also, for my reference, can you please comment here the link where this is documented?

Reply:
@onyb I took the value from ltcd: https://github.com/ltcsuite/ltcd/blob/master/wire/protocol.go#L150